### PR TITLE
Fix style issues

### PR DESCRIPTION
**Pull request by karawoo**
I believe this fixes all of the style issues caught by the linter in #64.

### CODE REVIEW

**Comment by tthyer** [comment]
This looks good to me, for what that's worth. You might want to add a pre-commit hook for this.

**Comment by karawoo** [comment]
Thanks @tthyer! Yeah I'm planning to look into adding a pre-commit hook as well.

File: R/app-server.R [hunk 1]
```diff
@@ -8,7 +8,9 @@ app_server <- function(input, output, session) {
     showModal(
       modalDialog(
         title = "Not logged in",
+        # nolint start
```
Comment (kdaily) [inline R/app-server.R]: Only a question - why use the `#nolint start/end` for a single line? I noted in other places you have a long line and did `#nolint` at the end. I see other places that use the `start/end` method for blocks of code, which makes sense.

Comment (karawoo) [reply R/app-server.R]: Because I made a mistake 😅 . I meant to use `# nolint` for single lines, and a block for anything that was more than one line.

**Comment by karawoo** [comment]
Thanks @kdaily! To explain my choices of linters:

* `object_length_linter(40)` -- this checks that the lengths of objects/functions aren't too long. Because some of my functions are S3 methods, they can be longer than recommended by default (e.g. `check_annotation_values.CsvFileTable`).
* `object_usage_linter = NULL` -- I disabled it for two reasons: 1) it complains about any function that's used without at `::` prefix. Such functions appear in test files, and in the app functions where I have imported all of shiny to avoid having to write `shiny::` a million times. 2) If an out of date version of the package is installed, it can create spurious errors about undefined functions in the up-to-date version of the package. So correct, exported functions cause errors because they didn't exist in whatever previous version of the package was installed.
* `object_name_linter = NULL` -- this complains about S3 methods and functions from other packages not being in snake_case. I can't do anything about functions from other packages using camelCase, nor is there a way to avoid using periods in S3 method definition, so I disabled it.
* `commented_code_linter = NULL` -- I disabled it because it's not that helpful and complains about double `##` for comments, which are common in the codebase because ESS uses them by default.

**Comment by kdaily** [comment]
All sensible to disable - thanks for taking the time to describe!

**Pull request merged**